scg-system/src/main/java/io/telicent/core/SmartCacheGraph.java

```diff
@@ -95,7 +95,7 @@ public static FusekiServer construct(String... args) {
                         rdfConfigPath.deleteOnExit();
 
                         try (FileOutputStream out = new FileOutputStream(rdfConfigPath)) {
-                            configModel.write(out, "TTL");
+                            configModel.write(out, "TTL", new File(configPath).getAbsoluteFile().getParentFile().toURI().toString());
```

Comment:
@karolina-telicent So this is the one line fix, basically take the directory in which the original config file sits and use it's `file:` URI as the Base URI passed into the Jena `write()` call, this adds an `@base` directive to the resulting Turtle file